.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,50 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: A recipe for reproducing the error. If possible, provide a link to a repository containing an executable repro scenario.
```

Comment:
The PR contains some irrelevant commits. I believe your PR should only contains the modification related to `echo` plugin and `rule_enabler.go`